lib/chrono_model.rb

```diff
@@ -17,6 +17,9 @@ module ChronoModel
   class Error < ActiveRecord::ActiveRecordError # :nodoc:
   end
 
+  # Validity is a tsrange field which has by default a precision of 6
```

Comment:
maybe `validity` is a trange column? Remove the concept of validity? Please use better terminology from postgres